environment.yml

```diff
@@ -34,7 +34,7 @@ dependencies:
   - packaging
   - pydata-sphinx-theme
   - pyyaml
-  - sphinx>=1.8.1,!=2.0.0
+  - sphinx>=7.1.1
```

Comment:
I dont think we should be pinning a sphinx that was released four days ago.

Reply:
https://matplotlib.org/stable/devel/min_dep_policy.html#test-and-documentation-dependencies

> We will support at least minor versions of the development dependencies released in the 12 months prior to our planned release.

Reply:
The highest we can pin this is ~ 5.0.0